easybuild/easyconfigs/m/memkind/memkind-1.5.0-GCCcore-5.4.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'memkind'
+version = '1.5.0'
+
+homepage = 'http://memkind.github.io'
+description = """User Extensible Heap Manager built on top of jemalloc which enables control of memory characteristics 
+and a partitioning of the heap between kinds of memory.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/memkind/memkind/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [
+    ('binutils', '2.26'),
+    ('Coreutils', '8.27'), # needed to ensure that ./build_jemalloc.sh works properly
+    ('Autotools', '20150215'), # needed to ensure that ./build_jemalloc.sh works properly
+]
+
+dependencies = [
+    ('tbb', '2017.4.174', '', True), # optional, to enable the tbb heap manager
+]
+
+# This can be also an EB dependency, but seems unnecesary
+osdependencies = [('numactl-devel', 'libnuma-devel ')]
```

Comment:
For easyconfigs included in the central repo, we should prefer using 'proper' dependencies rather than enforcing certain OS deps.

Sites that prefer certain OS deps can easily do so via `--filter-deps`